app/assets/javascripts/app/views/photos_view.js

```diff
@@ -7,20 +7,20 @@ app.views.Photos = app.views.InfScroll.extend({
 
     // viable for extraction
     this.stream.fetch();
-
-    this.setupLightbox();
     this.setupInfiniteScroll();
+    var self = this;
+    setTimeout(function(){
+      self.setupGallery();
+    }, 4000);
```

Comment:
@svbergerem : Can you help me with that? I don't manage to make this work unless I delay the setup.